workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -130,17 +126,27 @@ def calculate(self) -> None:
             elements=self.contribution_sum,
         )
 
-        if self.result.value >= self.threshold_yellow:
+        if self.result.value >= self.threshold_class_5:
```

Comment:
somewhere we need to add a space between these two substrings.

also in the indicator description

![image](https://user-images.githubusercontent.com/7045979/183708271-2df1620c-0e64-4b5e-a366-0e964416f2cf.png)